### PR TITLE
Update Set-CsTeamsCallingPolicy.md

### PR DESCRIPTION
**Pull request by SamChung-MS**
Adding a new Attribute for Call Recording.

### CODE REVIEW

**Review by dariomws — APPROVED** [review]
Just a small cosmetic detail.

I see that New-CsTeamsCallingPolicy is available but not documented. Does anyone know why?
image

**Comment by dariomws** [comment]
Remove-CsTeamsCallingPolicy is missing also.

File: skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md [hunk 1]
```diff
@@ -258,6 +258,20 @@ Type: String
...
+Determines whether cloud recording is allowed in a user's 1:1 call. Set this to TRUE to allow the user to be able to record 1:1 calls. Set this to FALSE to prohibit the user from recording 1:1 calls.
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Aliases:
+
 Required: False
 Position: Named
 Default value: None
```
Comment (get-itips) [inline skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md:277]:
```suggestion
Default value: False
```

Comment (get-itips) [reply skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md:277]: Looks like the default value is False (When not specifying this parameter)

**Review by get-itips — COMMENTED** [review]
just a small detail

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]
Once the changes are implemented, approved.

**Comment by yogkumgit** [comment]
@get-itips

**Comment by get-itips** [comment]
@SamChung-MS Please review the suggestions. Thanks

**Comment by SamChung-MS** [comment]
Looks good to me.

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks.

**Pull request merged**